packages/bundler-plugin-core/src/utils/providers/Bitbucket.ts

```diff
@@ -12,39 +12,61 @@ export function detect(envs: ProviderEnvs): boolean {
   return Boolean(envs?.CI) && Boolean(envs?.BITBUCKET_BUILD_NUMBER);
 }
 
-function _getBuild(inputs: ProviderUtilInputs): ProviderServiceParams["build"] {
+function _getBuild(
+  inputs: ProviderUtilInputs,
+  output: Output,
+): ProviderServiceParams["build"] {
   const { args, envs } = inputs;
   if (args?.build && args.build !== "") {
+    debug(`Using build: ${args.build}`, { enabled: output.debug });
     return args.build;
   }
-  return envs?.BITBUCKET_BUILD_NUMBER ?? null;
+  const build = envs?.BITBUCKET_BUILD_NUMBER ?? null;
+  debug(`Using build: ${build}`, { enabled: output.debug });
+  return build;
 }
 
-function _getBuildURL(): ProviderServiceParams["buildURL"] {
+function _getBuildURL(output: Output): ProviderServiceParams["buildURL"] {
   // TODO: https://github.com/codecov/uploader/issues/267
+  debug(`Using buildURL: ${null}`, { enabled: output.debug });
```

Comment:
I thought about this a little more and I take what I said back. I think when debugging, seeing that the value is `null` can just be helpful that our tool is using `null` for that value (and it can be told to a support person or cross-referenced in our docs as such). So you can ignore me on this!